client/my-sites/people/people-section-nav/index.jsx

```diff
@@ -1,22 +1,27 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	config = require( 'config' ),
-	find = require( 'lodash/find' ),
-	includes = require( 'lodash/includes' );
+import React from 'react';
+import config from 'config';
+import find from 'lodash/find';
+import includes from 'lodash/includes';
```

Comment:
We use [babel-plugin-lodash](https://github.com/lodash/babel-plugin-lodash) that does this for us so we can write:
```
import { find, includes } from 'lodash';
```